src/docquery/ext/transformers.py

```diff
@@ -58,17 +61,37 @@ def apply_tesseract(image: "Image.Image", lang: Optional[str], tesseract_config:
     height = [coord for idx, coord in enumerate(height) if idx not in irrelevant_indices]
 
     # turn coordinates into (left, top, left+width, top+height) format
-    actual_boxes = []
-    for x, y, w, h in zip(left, top, width, height):
-        actual_box = [x, y, x + w, y + h]
-        actual_boxes.append(actual_box)
+    actual_boxes = [[x, y, x + w, y + h] for x, y, w, h in zip(left, top, width, height)]
+
+    image_width, image_height = image.size
+
+    # finally, normalize the bounding boxes
+    normalized_boxes = [normalize_box(box, image_width, image_height) for box in actual_boxes]
+
+    assert len(words) == len(normalized_boxes), "Not as many words as there are bounding boxes"
+
+    return words, normalized_boxes
+
+
+
+def apply_easyocr(image: "Image.Image", reader):
```

Comment:
I appreciate the consistency here (i.e. placing this in the same file as `apply_ocr`) but this specific file is meant to code that's lifted out of the transformers repo for re-use. I'd prefer the equivalent `easy_ocr` stuff lives in a top-level package (outside of `ext`).

Reply:
To do this right way we probably should create a OCR processing class similar to document and be able to choose which one is used. Do you care if I pull the tesseract code out of `ext.transformer` and do this?

Reply:
That seems like a good idea. I'd be cool with moving the code out, but if you aren't planning to change it you could create a `TesseractOCR` class that _calls into_ that (if it's unchanged).